crates/sui-bridge-indexer/src/models.rs

```diff
@@ -22,9 +23,12 @@ pub struct TokenTransfer {
 pub struct TokenTransferData {
     pub chain_id: i32,
     pub nonce: i64,
-    pub sender_address: Vec<u8>,
+    pub block_height: i64,
+    pub timestamp_ms: i64,
+    pub txn_hash: Vec<u8>,
+    pub sender_address: String,
     pub destination_chain: i32,
-    pub recipient_address: Vec<u8>,
+    pub recipient_address: String,
```

Comment:
why string? We won't need to worry about encoding if we use bytes for addresses, ETH and Sui address are both Hex encoded but other chain might use different encoding

Reply:
in other schema we have in other tools we use strings as those are the easiest to copy and paste in a browser to check in an explorer. That makes sense to me.
Moreover the way I understand it, pure bytes run into the risk of making it impossible to export in format like csv, because a byte may be the representation of a comma (`,`).
In other words binary representation are difficult to deal with unless, I imagine, they are used programatically.
Thoughts?

Reply:
though... the indexer seems to like the byte array....